### PR TITLE
Failed the pipeline once there is error reported

### PR DESCRIPTION
**Pull request by sima-zhu**
It is currently only throwing a warning to pipeline when it detect new credential. Make changes so that it can fail the pipeline.

### CODE REVIEW

File: eng/pipelines/aggregate-reports.yml [hunk 1]
```diff
@@ -128,16 +128,15 @@ jobs:
         displayName: 'Run CredScan'
         inputs:
           suppressionsFile: 'eng\CredScanSuppression.json'
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml:131]: Why would you publish before the a analysis?

Comment (sima-zhu) [reply eng/pipelines/aggregate-reports.yml:131]: I am thinking to fail the post analysis if new issues coming. Then the publish step needs to either put before post analysis or run under condition of `successOrFailed`. Both two do not make too much difference, so I rearrange the order which is not blocking the publish step.

Comment (weshaggard) [reply eng/pipelines/aggregate-reports.yml:131]: That's fair. I agree it doesn't matter for our cases the order just seemed a little odd.

**Review by weshaggard — APPROVED** [review]

**Comment by jsquire** [comment]
Hi @sima-zhu.  There hasn't been recent engagement on this PR. Would you please be so kind as to let us know if this is still an active work stream by removing the no-recent-activity label? Otherwise, we'll close this out in 7 days.

**Comment by sima-zhu** [comment]
@jsquire Thanks for reminding! Closed the PR

**Pull request closed without merging**